.travis.yml

```diff
@@ -2,7 +2,6 @@ language: node_js
 node_js:
   - v7
   - v6
-  - v5
   - v4
```

Comment:
why not get rid of this one too?

Reply:
🤷 Honestly, didn't realize I had this committed. A lot of these repos use super old versions, and _odd_ version numbers so they never had LTS. I'm trying not to muck with them, but if the travis builds are being honored (there is evidence in the commit history of workikng towards successful builds), then I'll work on it. Otherwise, and this applies to most of these repos so far, I just ignore the build failing because loads of previous commits/builds have failed and never been addressed.